locale/ar_IQ/countries.xml

```diff
@@ -8,11 +8,7 @@
   * Copyright (c) 2000-2017 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
-  * Localized list of countries.
-  * Please don't edit or translate. This file is automatically generated using
-  * the ISO 3166 files of Debian's iso-codes package
-  * (https://packages.debian.org/sid/all/iso-codes) using the
-  * tools/poToCountries.php tool.
+  * Localized list of countries. Based on ISO 3166-1:2006.
```

Comment:
We think it's best to not edit this file directly as we generate it automatically from the Debian country list. Could you revert the changes to this file?

Reply:
It is OK if the automatically generated list is in Arabic too, otherwise it will be solely in English.
By the way, this file had been translated since OJS 3.0.0.0
I think it's your call. You may do what's best for the system.